Windows compilation error - error C2027: use of undefined type 'Kratos::Variable<Kratos::Flags>'

**Description**
Local windows compilation of the master throws the error below although CI windows compilation passes. Local windows compilation of a 1month old branch works fine.

**Scope**
Where in Kratos is this bug happening. 
- Compilation


**To Reproduce**
configure.bat

```
@echo off
rem Please do not modify this script

rem For any question please contact with us in:
rem  - https://github.com/KratosMultiphysics/Kratos

rem Optional parameters:
rem You can find a list will all the compiation options in INSTALL.md or here:
rem  - https://github.com/KratosMultiphysics/Kratos/wiki/Compilation-options

rem Set compiler
set CC=cl.exe
set CXX=cl.exe

rem Set variables
set KRATOS_SOURCE=~0,-1%/..
set KRATOS_BUILD=%KRATOS_SOURCE%/build
set KRATOS_APP_DIR=applications

set STRUCTURAL_DISABLE_ADVANCED_CONSTITUTIVE_LAWS=ON

rem Warning: In windows this option only works if you run through a terminal with admin privileges
rem set KRATOS_INSTALL_PYTHON_USING_LINKS=ON

rem Set basic configuration
set KRATOS_BUILD_TYPE=Debug
set BOOST_ROOT=C:\boost_1_67_0
set PYTHON_EXECUTABLE=C:\Python38\python.exe

rem Set applications to compile
set KRATOS_APPLICATIONS=
rem CALL :add_app %KRATOS_APP_DIR%\ParticleMechanicsApplication;
CALL :add_app %KRATOS_APP_DIR%\MappingApplication;
rem CALL :add_app %KRATOS_APP_DIR%\StructuralMechanicsApplication;
rem CALL :add_app %KRATOS_APP_DIR%\CoSimulationApplication;
rem CALL :add_app %KRATOS_APP_DIR%\MappingApplication;

rem Clean
del /F /Q "%KRATOS_BUILD%\%KRATOS_BUILD_TYPE%\cmake_install.cmake"
del /F /Q "%KRATOS_BUILD%\%KRATOS_BUILD_TYPE%\CMakeCache.txt"
del /F /Q "%KRATOS_BUILD%\%KRATOS_BUILD_TYPE%\CMakeFiles"

rem Configure
@echo on
cmake -G"Visual Studio 16 2019" -H"%KRATOS_SOURCE%" -B"%KRATOS_BUILD%\%KRATOS_BUILD_TYPE%"          ^
-DINCLUDE_FEAST=OFF                                                                                 ^
-DLAPACK_LIBRARIES="C:\external_libraries\liblapack.lib"                                         ^
-DKRATOS_BUILD_TESTING=OFF                                                                       ^
-DSTRUCTURAL_DISABLE_ADVANCED_CONSTITUTIVE_LAWS=ON                                         ^
-DUSE_EIGEN_MKL=OFF                                         ^
-DBLAS_LIBRARIES="C:\external_libraries\libblas.lib"

rem Build
cmake --build "%KRATOS_BUILD%/%KRATOS_BUILD_TYPE%" --target install -- /property:configuration=%KRATOS_BUILD_TYPE% /p:Platform=x64
goto:eof

rem Function to add apps
:add_app
set KRATOS_APPLICATIONS=%KRATOS_APPLICATIONS%%1;
goto:eof
```

Console output

```
C:\Users\s4549408\Documents\Kratos_2\scripts>debug_2.bat
The system cannot find the file specified.
The system cannot find the file specified.
The system cannot find the file specified.

C:\Users\s4549408\Documents\Kratos_2\scripts>cmake -G"Visual Studio 16 2019" -H"~0,-1/.." -B"~0,-1/../build\Debug"          -DINCLUDE_FEAST=OFF                                                                                 -DLAPACK_LIBRARIES="C:\external_libraries\liblapack.lib"                                         -DKRATOS_BUILD_TESTING=OFF                                                                       -DSTRUCTURAL_DISABLE_ADVANCED_CONSTITUTIVE_LAWS=ON                                         -DUSE_EIGEN_MKL=OFF                                         -DBLAS_LIBRARIES="C:\external_libraries\libblas.lib"
-- Selecting Windows SDK version 10.0.18362.0 to target Windows 10.0.14393.
-- No KRATOS_SOURCE_DIR is defined, using: C:/Users/s4549408/Documents/Kratos_2
-- No KRATOS_BINARY_DIR is defined, using: C:/Users/s4549408/Documents/Kratos_2/build/Debug
-- No INSTALL_RUNKRATOS is defined, setting to ON
-- No KRATOS_INSTALL_PYTHON_USING_LINKS is defined, setting to OFF
-- No CMAKE_INSTALL_MESSAGE is defined. Setting to NEVER
-- CMAKE_SYSTEM_NAME = Windows
Detected compiler as MSVC
-- User defined install dir C:/Users/s4549408/Documents/Kratos_2/bin/Debug
"KRATOS_SHARED_MEMORY_PARALLELIZATION" not defined, defaulting to "OpenMP"
Using OpenMP for shared memory parallelization
-- Boost Include: C:/boost_1_67_0
-- Boost Linkdir:
-- Could NOT find ZLIB (missing: ZLIB_INCLUDE_DIR)
-- Preparing local ZLIB compilation.
-- AMatrix not found. Please define the AMATRIX_DIR pointing to the AMatrix root directory.

CMAKE_BUILD_TYPE ........ Debug

WARNING: YOUR GIDPOST IS BEING COMPILED  WITHOUT DEBUG (ALL ASSERTS WILL BE AVOIDED)
-- Configuring applications (ENV):
Adding application 'applications\MappingApplication'
**** configuring KratosMappingApplication ****

***********************************************************************

        LIST OF APPLICATIONS THAT ARE GOING TO BE COMPILED:

        MappingApplication

        APPLICATIONS ADDED TO COMPILATION THROUGH DEPENDENCIES:


***********************************************************************

List of upcoming api-breaking changes:
         Making variables non-copiable: https://github.com/KratosMultiphysics/Kratos/pull/6339
         Deprecation of NOT_FLAGS : https://github.com/KratosMultiphysics/Kratos/pull/6233
         Removing long deprecated conditions : https://github.com/KratosMultiphysics/Kratos/pull/6475
         Changing 'FindNodalNeighboursProcess' constructor signature: https://github.com/KratosMultiphysics/Kratos/pull/4947
         Minor update of import mechanism for applications: https://github.com/KratosMultiphysics/Kratos/pull/6556
         Update of Element, Condition and Scheme Interface: https://github.com/KratosMultiphysics/Kratos/issues/7381
         VariableComponent and VectorComponentAdaptor are no longer used: https://github.com/KratosMultiphysics/Kratos/pull/6686
-- triangle : C:/Users/s4549408/Documents/Kratos_2/external_libraries/triangle
python libraries to be installed are: C:/Python38/libs/python38.lib
-- Configuring done
-- Generating done
-- Build files have been written to: C:/Users/s4549408/Documents/Kratos_2/build/Debug

C:\Users\s4549408\Documents\Kratos_2\scripts>rem Build

C:\Users\s4549408\Documents\Kratos_2\scripts>cmake --build "~0,-1/../build/Debug" --target install -- /property:configuration=Debug /p:Platform=x64
Microsoft (R) Build Engine version 16.4.0+e901037fe for .NET Framework
Copyright (C) Microsoft Corporation. All rights reserved.

  kratos_version.cpp
  KratosVersion.vcxproj -> C:\Users\s4549408\Documents\Kratos_2\build\Debug\kratos\KratosVersion.dir\Debug\KratosVersio
  n.lib
  zlib.vcxproj -> C:\Users\s4549408\Documents\Kratos_2\build\Debug\external_libraries\zlib\Debug\zlibd.dll
  gidpost.vcxproj -> C:\Users\s4549408\Documents\Kratos_2\build\Debug\external_libraries\gidpost\Debug\gidpost.lib
  triangle.vcxproj -> C:\Users\s4549408\Documents\Kratos_2\build\Debug\external_libraries\triangle\Debug\triangle.lib
  parallel_environment.cpp
C:\Users\s4549408\Documents\Kratos_2\kratos\includes/kratos_components.h(84,1): error C2027: use of undefined type 'Kra
tos::Variable<Kratos::Flags>' [C:\Users\s4549408\Documents\Kratos_2\build\Debug\kratos\KratosCore.vcxproj]
C:\Users\s4549408\Documents\Kratos_2\kratos\includes/kratos_components.h(490): message : see declaration of 'Kratos::Va
riable<Kratos::Flags>' [C:\Users\s4549408\Documents\Kratos_2\build\Debug\kratos\KratosCore.vcxproj]
C:\Users\s4549408\Documents\Kratos_2\kratos\includes/kratos_components.h(81): message : while compiling class template
member function 'void Kratos::KratosComponents<Kratos::Variable<Kratos::Flags>>::Add(const std::string &,const TCompone
ntType &)' [C:\Users\s4549408\Documents\Kratos_2\build\Debug\kratos\KratosCore.vcxproj]
          with
          [
              TComponentType=Kratos::Variable<Kratos::Flags>
          ]
C:\Users\s4549408\Documents\Kratos_2\kratos\includes/kratos_components.h(490): message : see reference to class templat
e instantiation 'Kratos::KratosComponents<Kratos::Variable<Kratos::Flags>>' being compiled [C:\Users\s4549408\Documents
\Kratos_2\build\Debug\kratos\KratosCore.vcxproj]
  periodic_condition.cpp
  vtk_output.cpp
  Generating Code...

C:\Users\s4549408\Documents\Kratos_2\scripts>goto:eof
```


**Environment**
- OS: Windows 10
- Branch: master
- Python 3.8

--- a/kratos/includes/kratos_components.h
+++ b/kratos/includes/kratos_components.h
@@ -24,9 +24,10 @@
 
 // Project includes
 #include "includes/define.h"
-#include "containers/flags.h"
-#include "utilities/quaternion.h"
 #include "includes/data_communicator.h"
+#include "containers/flags.h"
+#include "containers/variable.h"
+#include "utilities/quaternion.h"
 
 namespace Kratos
 {

